context/app/static/js/helpers/functions.ts

```diff
@@ -176,3 +176,11 @@ export function filterObjectByKeys<O extends object, K extends keyof O>(obj: O,
 export function getOriginSamplesOrgan(entity: { origin_samples_unique_mapped_organs: string[] }) {
   return entity.origin_samples_unique_mapped_organs.join(', ');
 }
+
+export function generateCommaList(list: string[]): string {
```

Comment:
Since this is meant for re-use, it would be nice to add a spec or two.

Reply:
I added a JSDoc - if there's another preferred documentation format though, let me know!

Reply:
I was only asking for a test or two, but this is extra helpful!